test/integration/targets/nxos_lacp/tests/cli/merged.yaml

```diff
@@ -6,21 +6,38 @@
   nxos_feature:
     feature: lacp
 
+- set_fact:
+    mac:
+      address: 00c1.4c00.bd15
+      role: primary
+  when: platform is search('N9K') and imagetag is not search('I2|I4|I5|I6')
+
 - block:
   - name: Merged
     nxos_lacp: &merged
       config:
         system:
           priority: 11
+          mac: "{{ mac|default(omit) }}"
       state: merged
     register: result
 
+  - assert:
+      that:
+        - "result.before|length == 0"
+        - "result.changed == true"
+        - "'lacp system-priority 11' in result.commands"
+        - "'lacp system-mac 00c1.4c00.bd15 role primary' in result.commands"
+        - "result.commands|length == 2"
+    when: platform is search('N9K') and imagetag is not search('I2|I4|I5|I6')
```

Comment:
Thanks for adding the `imagetag` check.  Go ahead and add `I3` to this list as well.   We don't test this in our CI but we should include all tags that are not supported.  Also, I believe you can jus do this `search('I[2-6]')` but test it to make sure :)

Reply:
Yeah, regex works.